website/docs/getting-started/intro.md

```diff
@@ -50,5 +74,6 @@ describe('App', () => {
   });
 });
 ```
+:::tip It's that simple! 😱
+:::
 
-You also need to start the Jest Preview Server by running the CLI `jest-preview`. Please continue to read [Usage](/docs/getting-started/usage) for the details instructions.
```

Comment:
I have a suggestion here. Since the simple instruction above does not support CSS and viewing images yet. So we mention something like "To make it look even better, follow the Installation guide"

Reply:
Added Installation guide link as per your suggestion